src/prefect/tasks/databricks/models.py

```diff
@@ -75,7 +75,7 @@ class NewCluster(BaseModel, extra=Extra.allow):
     aws_attributes: Optional[AwsAttributes] = None
     driver_node_type_id: Optional[str] = None
     ssh_public_keys: Optional[List[str]] = None
-    custom_tags: Optional[str] = None
+    custom_tags: Optional[Json] = None
```

Comment:
Is it possible to make this a `Dict[str, Any]` type? I think that typing would be more intuitive for users and align with other similar fields like `spark_conf` on the `NewCluster` model.

Reply:
@desertaxle good catch. I did it, the tests are failing, any idea?